libcontainerd/client_windows.go

```diff
@@ -299,7 +299,7 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
 		Owner:         defaultOwner,
 		TerminateOnLastHandleClosed: true,
 		HvRuntime: &hcsshim.HvRuntime{
-			ImagePath:       `c:\program files\lcow`,
+			ImagePath:       `c:\Program Files\Linux Containers`,
```

Comment:
Yeah, all this is going to go away shortly anyway, but I will address in a follow-up. The change of directory name is the important bit for now.